beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperExpiredMetadataSchedulerApiaryIntegrationTest.java

```diff
@@ -0,0 +1,251 @@
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.awaitility.Duration;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.PurgeQueueRequest;
+import com.amazonaws.services.sqs.model.SendMessageRequest;
+
+import com.expediagroup.beekeeper.core.model.HousekeepingMetadata;
+import com.expediagroup.beekeeper.integration.model.AddPartitionSqsMessage;
+import com.expediagroup.beekeeper.integration.model.AlterPartitionSqsMessage;
+import com.expediagroup.beekeeper.integration.model.AlterTableSqsMessage;
+import com.expediagroup.beekeeper.integration.model.CreateTableSqsMessage;
+import com.expediagroup.beekeeper.integration.utils.ContainerTestUtils;
+import com.expediagroup.beekeeper.scheduler.apiary.BeekeeperSchedulerApiary;
+
+@Testcontainers
+public class BeekeeperExpiredMetadataSchedulerApiaryIntegrationTest extends BeekeeperIntegrationTestBase {
+
+  private static final int TIMEOUT = 5;
+  private static final String QUEUE = "apiary-receiver-queue";
+  private static final String SCHEDULED_EXPIRED_METRIC = "metadata-scheduled";
+  private static final String HEALTHCHECK_URI = "http://localhost:8080/actuator/health";
+  private static final String PROMETHEUS_URI = "http://localhost:8080/actuator/prometheus";
+
+  private static final String PARTITION_KEYS = "{ \"event_date\": \"date\", \"event_hour\": \"smallint\"}";
+  private static final String PARTITION_A_VALUES = "[ \"2020-01-01\", \"0\" ]";
+  private static final String PARTITION_B_VALUES = "[ \"2020-01-01\", \"1\" ]";
+  private static final String PARTITION_A_NAME = "event_date=2020-01-01/event_hour=0";
+  private static final String PARTITION_B_NAME = "event_date=2020-01-01/event_hour=1";
+  private static final String LOCATION_A = "s3://location-a";
+  private static final String LOCATION_B = "s3://location-b";
+
+  @Container
+  private static final LocalStackContainer SQS_CONTAINER = ContainerTestUtils.awsContainer(SQS);
+  private static AmazonSQS amazonSQS;
+
+  @BeforeAll
+  public static void init() {
+    String queueUrl = ContainerTestUtils.queueUrl(SQS_CONTAINER, QUEUE);
+    System.setProperty("properties.apiary.queue-url", queueUrl);
```

Comment:
Should we not unset this in the `@AfterAll` method so as not to "infect" other tests?

Reply:
Ah I never thought of that. I will do that